src/config/settings.ts

```diff
@@ -441,6 +478,14 @@ const settings: Settings = {
         subgraphUrl: <string>process.env['SOVRYN_SUBGRAPH_API'],
       },
     },
+    [DexProtocolName.UNISWAP_V3]: {
```

Comment:
for liquidity we have `[chain][protocol]` and here we have `[protocol][chain]`, let's unify all to `[chain][protocol]`.

Reply:
Let's do separately